tutor/src/helpers/pardot.ts

```diff
@@ -0,0 +1,66 @@
+import { isProd } from './production'
+import { DOMWindow } from 'jsdom'
+
+declare global {
+    interface Window {
+        piAId: string
+        piCId: string
+        piHostname: string
+    }
+}
+
+const SANDBOX_HOSTNAME = /dev.tutor.sandbox.openstax.org/
+
+export const pardotConfig = {
+    sand: {
+        piAId: '309222',
+        piCId: '2313',
+        piHostname: 'pi.demo.pardot.com',
+    },
+    prod: {
+        piAId: '219812',
+        piCId: '1120',
+        piHostname: 'pi.pardot.com',
+    },
+};
+
+
+export default class Pardot {
+
+    static getConfig(hostname: string) {
+        if (this.isSandbox(hostname)) { return pardotConfig.sand }
+
+        return this.isProduction ? pardotConfig.prod : pardotConfig.sand
+    }
+
+    static get isProduction() {
+        return isProd
```

Comment:
does isProd return true in qa/staging?

Reply:
Yeah :/ that's why I'm checking hostname instead. Long term, would be nice to have a sandbox/qa/staging env configs.

Reply:
does this logic use the prod config in staging? do we want that?

Reply:
I'm glad you pointed that out, I'm guessing we probably don't want that, just like we didn't want it for sandbox. I really should make this do the opposite and only use production if it matches production host, I'll update this PR with that 👍

Reply:
Updated to check for production hostname instead, which will let us default to the demo config on sandbox, qa, etc: https://github.com/openstax/tutor-js/pull/3784/commits/3a977478f2b5bae1b1a485b8243437ae7eaa8ad4